packages/space-opera/src/components/mobile_view/types.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {GltfModel} from '@google/model-viewer-editing-adapter/lib/main';
+
+export const DOMAIN = 'https://piping.glitch.me/';
+
+export interface URLs {
+  gltf: string|undefined;
+  usdz: string|undefined;
+  env: string|undefined;
+}
+
+export interface MobileSession {
+  id: number;
+  isStale: boolean;
+  os: string;
+}
+
+export interface EditorUpdates {
+  gltfChanged: boolean;
+  gltfId: number;
+  iosChanged: boolean;
+  usdzId: number;
+  stateChanged: boolean;
+  envChanged: boolean;
+  envIsHdr: boolean;
+}
+
+export interface MobilePacket {
+  updatedContent: EditorUpdates;
+  snippet?: any;
+}
+
+export function getRandomInt(max: number): number {
+  return Math.floor(Math.random() * Math.floor(max));
```

Comment:
These don't really look like types. Maybe utilities?

Reply:
Yeah, I was split between making a new file for them or leaving them in with types.